modules/op-creating-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -10,16 +10,28 @@ To run pipelines using {pac}, you can create pipeline run definitions or templat
 
 The {pac} resolver bundles the pipeline runs with all tasks as a single pipeline run without external dependencies.
 
+A {pac} pipeline run has the same format as any other pipeline run. However, it also includes the following additional features:
+
+* You can use parameters in the pipeline run specification. When {pac} creates a pipeline run from the definition, it replaces the parameters with the values that they represent.
+
+* You can use annotations to match the pipeline run definition to events. When the defined event happens, {pac} creates and starts a pipeline run based on the definition.
+
+* You can use annotations to define whether {pac} automatically cancels the pipeline run in certain cases.
+
+
 [NOTE]
 ====
 * For pipelines, use at least one pipeline run with a spec, or a separated `Pipeline` object.
 * For tasks, embed task spec inside a pipeline, or define it separately as a Task object.
 ====
 
+[id="parameters-creating-pipeline-run-using-pipelines-as-code_{context}"]
+== Using parameters in a pipeline run specification
```

Comment:
There are multiple modules within this reference module, including some that should be procedures.
 
> A module should not contain another module. 

https://redhat-documentation.github.io/modular-docs/#_creating_modules

Reply:
This is a reference module, but I guess splitting it up won't really hurt as long as I keep the IDs, so I'll do that.